Gemfile.lock

```diff
@@ -209,60 +209,60 @@ GEM
       sprockets-rails
       tilt
     semantic_range (2.3.0)
-    spring (2.1.0)
+    spring (2.1.1)
     spring-watcher-listen (2.0.1)
       listen (>= 2.7, < 4.0)
       spring (>= 1.2, < 3.0)
     sprockets (4.0.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
-    sprockets-rails (3.2.1)
+    sprockets-rails (3.2.2)
       actionpack (>= 4.0)
       activesupport (>= 4.0)
       sprockets (>= 3.0.0)
     thor (1.0.1)
     thread_safe (0.3.6)
     tilt (2.0.10)
-    tinymce-rails (5.4.1)
+    tinymce-rails (5.6.1)
       railties (>= 3.1.1)
-    trestle (0.9.3)
+    trestle (0.9.4)
       activemodel (>= 4.2.0)
       kaminari (>= 1.1.0)
       railties (>= 4.2.0)
       sprockets-rails (>= 2.0.0)
-    trestle-active_storage (3.0.0)
+    trestle-active_storage (3.0.1)
       rails (>= 5.2, < 7)
       trestle (~> 0.9.0, >= 0.9.3)
-    trestle-auth (0.4.0)
+    trestle-auth (0.4.1)
       bcrypt (~> 3.1.7)
-      trestle (~> 0.9.0)
+      trestle (~> 0.9.0, >= 0.9.3)
     trestle-tinymce (0.3.0)
       tinymce-rails (~> 5.1)
       trestle (~> 0.9.0)
     turbolinks (5.2.1)
       turbolinks-source (~> 5.2)
     turbolinks-source (5.2.0)
-    tzinfo (1.2.7)
+    tzinfo (1.2.8)
       thread_safe (~> 0.1)
     unf (0.1.4)
       unf_ext
     unf_ext (0.0.7.7)
-    warden (1.2.8)
-      rack (>= 2.0.6)
-    web-console (4.0.4)
+    warden (1.2.9)
+      rack (>= 2.0.9)
+    web-console (4.1.0)
```

Comment:
### [web-console](https://github.com/rails/web-console)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 4.0.4 → 4.1.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **13** commits. See the full changes on [the compare page](https://github.com/rails/web-console/compare/v4.0.4...v4.1.0).

These are the first 10 commits:
- (ace938e) [Use MutationObserver instead of Mutation Events](https://github.com/rails/web-console/commit/ace938efad7011826c57b000c1e3e818e6edde55)
- (a61dd15) [Prevent warnings by removing template formats](https://github.com/rails/web-console/commit/a61dd155677b42e42fc1694de23d31c3e74e9291)
- (bc5d230) [Add CSP nonce to injected scripts and styles](https://github.com/rails/web-console/commit/bc5d230b8a97c066a5ad27541fccb94d0df36f27)
- (28dfbc4) [Merge pull request #296 from mikelkew/csp-nonce](https://github.com/rails/web-console/commit/28dfbc4119f3cde860e494fca63fbac161795dbf)
- (008e6c6) [Merge pull request #297 from mikelkew/mutation-observer](https://github.com/rails/web-console/commit/008e6c624cfa9ed2d4b7da2c35ef39fa4bd1e6f1)
- (eeaf847) [Merge pull request #298 from mikelkew/deprecated-template-na…](https://github.com/rails/web-console/commit/eeaf8473d9706999b11b381a0637574f3e603b9d)
- (676613b) [Fix scrolling in web console terminal](https://github.com/rails/web-console/commit/676613b713a96e9d9b7cc7fb19ecca7192fa086a)
- (788eeba) [Merge pull request #302 from amikishiyev/fix-terminal](https://github.com/rails/web-console/commit/788eebadec08743c9e4446710921ac70e5b60fac)
- (bb7ec81) [Instantiate view with empty template cache](https://github.com/rails/web-console/commit/bb7ec818009c0fadb227446ad7009440d3d678df)
- (5451b57) [Merge pull request #304 from stephannv/fix/instantiate_view_…](https://github.com/rails/web-console/commit/5451b578e838ae0178f0bde1bcd36ac1bfcb717e)


</details>